app/src/lib/hostedServices/github/octokit.ts

```diff
@@ -0,0 +1,9 @@
+import { Octokit } from '@octokit/rest';
+
+export function octokitFromAccessToken(accessToken: string | undefined) {
+	return new Octokit({
+		auth: accessToken,
+		userAgent: 'GitButler Client',
+		baseUrl: 'https://api.github.com'
+	});
+}
```

Comment:
When do we want to pass undefined as the accesss token?

Reply:
Never!